components/button-group.scss

```diff
@@ -0,0 +1,5 @@
+$fd-icons-path: "../scss/icons/";
+
+@import "../scss/components/button";
+@import "../scss/components/button-group";
+@import "../scss/icons/icon";
```

Comment:
This is another component that is a little strange to apply self-contained styles to because the majority of the style content loaded here is for the button component, which has its own self-contained styles. I know these were added because of the playground examples, but honestly the playground examples should be reworked to not include any specific fundamental-related content.  Although this is the button group component, the examples shouldn't use the button component for the children.  The essence of a button group is a wrapper around multiple elements (typically buttons although the CSS is written such that it wouldn't have to be) that have rounded corners and borders to make sure they format correctly when in a group.  The playground examples can show a `<button>` element, but shouldn't use any of the `fd-button` classes.  They could be styled with some custom CSS on the playground page itself using a temp class variable name (_e.g._ `btn-playground`).

On the main docs site, we can keep the buttons in place as the children as those are more use case-related.

Reply:
@greg-a-smith If I understand correctly, I should remove the `fd-button` class and add some custom class called `btn-playground` which has the same style as the fd-button? Where would I put this new class? Is there a file where we put custom sass in the docs? Or maybe just in `<style>` tags?